libobs/obs-output.c

```diff
@@ -888,14 +901,8 @@ obs_encoder_t *obs_output_get_audio_encoder(const obs_output_t *output,
 	if (!obs_output_valid(output, "obs_output_get_audio_encoder"))
 		return NULL;
 
-	if ((output->info.flags & OBS_OUTPUT_MULTI_TRACK) != 0) {
-		if (idx >= MAX_AUDIO_MIXES) {
-			return NULL;
-		}
-	} else {
-		if (idx > 0) {
-			return NULL;
-		}
+	if (idx >= MAX_OUTPUT_AUDIO_ENCODERS) {
+		return NULL;
 	}
```

Comment:
maybe shouldn't use braces?

Reply:
Fixed in 6333146988f2f6f2b5558bd2e129b691f9b5cb85